### PR TITLE
feat(probe-image-size): v7.2 update options & return types

### PR DESCRIPTION
**Pull request by yashafromrussia**
This change has been introduced in `probe-image-size` 2 years ago, in v6

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/nodeca/probe-image-size#probesrc--optionskeepopen---promise
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## probe-image-size ([<kbd>unpkg</kbd>](https://unpkg.com/browse/probe-image-size@latest/))
was missing the following properties:
1. parsers
1. Error
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 24653b3e4bfd19a5bc2165230e18a44f9b9450bc
</p>

**Comment by peterblazejewicz** [comment]
sould we bump to v7.2?

**Comment by yashafromrussia** [comment]
> sould we bump to v7.2?

@peterblazejewicz thank you! I updated the version 👍

File: types/probe-image-size/tslint.json [hunk 1]
```diff
@@ -1 +1,6 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-empty-interface": false
+    }
```
Comment (peterblazejewicz) [inline types/probe-image-size/tslint.json:5]:
```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```
this should be updated on really demanding use cases. Inline comment should work

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@yashafromrussia thanks!

**Comment by yashafromrussia** [comment]
Ready to merge

**Pull request merged**